frontend/src/ts/menu.ts

```
@@ -9,7 +9,22 @@ let nextImageMenuButton: HTMLElement;
 
 let imageOverlayVisible: boolean = false;
+let linkOverlayVisible: boolean = false;
 
+const redirectsContainer = document.getElementById(
+  "redirects-container",
+) as HTMLElement | null;
+let redirects: NodeListOf<HTMLAnchorElement> | null;
+let currentRedirectIndex = -1;
+
+let allowMoreInfo: boolean;
+let infoKeyPress: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)